genai-on-vertex-ai/retrieval_augmented_generation/diy_rag_with_vertexai_apis/build_grounded_rag_app_with_vertex.ipynb

```diff
@@ -1006,7 +1006,7 @@
     "    if not CREATE_RESOURCES:\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.        bucket = storage_client.create_bucket(bucket_name, project=PROJECT_ID)
</pre><p>Any reason why <code>PROJECT_ID</code> is required arg ? From the docs - <a href="https://cloud.google.com/storage/docs/creating-buckets#create_a_new_bucket" target="_blank">https://cloud.google.com/storage/docs/creating-buckets#create_a_new_bucket</a> it doesn't seem like it is required. </p><p> </p><p>Also since we have an authed session, it should pick up the default project ID from there anyway.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
It's been too long since I tried it... just remembered it not working without this line.

Reply:
Yeah I can rerun the notebook today and check. We can close the PR if it works :)